tools/utils.ts

```diff
@@ -15,14 +15,19 @@ export const getPackageInfo = async (): Promise<Package[]> => {
   const packages: Package[] = [];
 
   for (const subDir of await fs.readdir(PACKAGES_DIR)) {
-    for (const packageDir of await fs.readdir(path.resolve(PACKAGES_DIR, subDir))) {
-      const packagePath = path.resolve(PACKAGES_DIR, subDir, packageDir);
-      const pkg = await fs.readJson(path.resolve(packagePath, 'package.json'));
-      packages.push({
-        path: packagePath,
-        name: pkg.name,
-        manifest: pkg,
-      });
+    const subDirPath = path.resolve(PACKAGES_DIR, subDir);
```

Comment:
We used to assume that anything in the `packages/` folder was a folder as well. Here, I add a little `isDirectory()` check to make sure that we don't assume `package/foo` is a folder.